src/context/liked-context.js

```diff
@@ -4,7 +4,7 @@ import { likedReducer } from "../reducer";
 const LikedContext = createContext();
 
 const LikedProvider = ({ children }) => {
```

Comment:
You can change the name of context as it does not represent other playlist which are present in this context

Reply:
updated, thank you ravi